build/Dockerfile

```
@@ -1,12 +1,46 @@
-ARG BASE_IMAGE_TAG
-FROM neuvector/adapter_base:${BASE_IMAGE_TAG}
+#
+# Builder image
+FROM registry.suse.com/bci/golang:1.22.7 AS builder
```

Either lean on a higher level mutable tag, or add `renovate`/`updatecli` to ensure this is being kept up to date.

```suggestion
FROM registry.suse.com/bci/golang:1.22 AS builder
```

---

@jayhuang-suse any concern about this? I don't think Golang would bring breaking changes in minor update, so I'm okay with using 1.22.